### PR TITLE
Update OVN-Kubernetes default network policy test scenario

### PR DESCRIPTION
**Pull request by varad-ahirwadkar**
Fixing the issue https://github.com/ocp-power-automation/ocp4-playbooks-extras/issues/5 and refactored the code for default network policy

Signed-off-by: Varad Ahirwadkar <varad.ahirwadkar@ibm.com>

### CODE REVIEW

File: playbooks/roles/ocp-ovnkube-cni/tasks/networkpolicy.yml [hunk 1]
```diff
@@ -245,97 +245,108 @@
...
+            name: cluster
+          spec: 
+            projectRequestTemplate: null
 
-    - lineinfile:
-        line: "Default NetworkPolicy Success"
+    - name: Save success result
+      lineinfile:
+        state: present
+        line: "Default network policy has successfully added to the new project"
```
Comment (pravin-dsilva) [inline playbooks/roles/ocp-ovnkube-cni/tasks/networkpolicy.yml:338]:
```suggestion
        line: "Default network policy has been successfully added to the new project"
```

File: playbooks/roles/ocp-ovnkube-cni/tasks/networkpolicy.yml [hunk 2]
```diff
@@ -245,97 +245,108 @@
...
-        - create_template.failed == false and create_config.failed == false
-        - confim_op.failed == false
+      when: not network_policy.failed
 
-    - lineinfile:
-        line: "Default NetworkPolicy Failed {{ item.error }}"
+    - name: Save failure result
+      lineinfile:
+        state: present
+        line: "Default network policy has not added to the new project"
```
Comment (pravin-dsilva) [inline playbooks/roles/ocp-ovnkube-cni/tasks/networkpolicy.yml:346]:
```suggestion
        line: "Fail: Default network policy has not been added to the new project."
```

**Review by pravin-dsilva — CHANGES_REQUESTED** [review]

File: playbooks/roles/ocp-ovnkube-cni/tasks/networkpolicy.yml [hunk 3]
```diff
@@ -245,97 +245,108 @@
...
 
-    - name: Delete template
-      shell: oc delete -f "{{ ovnkube_setup_directory }}/template.yaml" -n openshift-config
-      ignore_errors: yes
+    - name: Check if the project gets deleted
+      shell: oc get project | grep test-default-network-policy | wc -l
+      register: test_project
+      until: test_project.stdout|int == 0
+      retries: 15
+      delay: 30
```
Comment (yussufsh) [inline playbooks/roles/ocp-ovnkube-cni/tasks/networkpolicy.yml:313]: @varad-ahirwadkar I think wait is available in k8s module for deletion on all resources https://docs.ansible.com/ansible/latest/collections/kubernetes/core/k8s_module.html#parameter-wait.
Could you please try combining these two tasks into one? We won't need oc shell commands everywhere.

Comment (varad-ahirwadkar) [reply playbooks/roles/ocp-ovnkube-cni/tasks/networkpolicy.yml]: Sure. I have combined these two tasks.

**Review by yussufsh — APPROVED** [review]
/lgtm

**Pull request merged**